dojo/tools/api_sonarqube/api_client.py

```diff
@@ -261,21 +261,31 @@ def get_issue(self, issue_key):
             if issue["key"] == issue_key:
                 return issue
         raise Exception(
-            f"""Expected Issue "{issue_key}", but it returned "
-            "{[x.get('key') for x in response.json().get('issues')]}."""
+            f"""Expected Issue "{issue_key}", but it returned
```

Comment:
Removing`"` at the end of the line might be problematic. Do you think it's working well now?

Reply:
I extended the exception output as I wanted to see the whole output from the SonarQube API Call. Sometimes still an error comes up because of an empty call coming back. But It doesn't have an effect on the findings. If you have any suggestions please share them...

Reply:
I have no problem with extending the error message (it might be really useful). Just wasn't sure about formating.

Not even sure if `f"""` was the best idea.

What do you think about 
```python
f'Expected Issue "{issue_key}", but it returned' \
f"{[x.get('key') for x in response.json().get('issues')]}. " \
"Full response: " \
f"{response.json()}"
```

Reply:
got you. Yeah we may adapt it. But backslash doesn't make it better (and makes flake8 angry).

Maybe we can do it like:

`f"Expected Issue \"{issue_key}\", but it returned"
f"{[x.get('key') for x in response.json().get('issues')]}. "
"Full response: "
f"{response.json()}"`

(already pushed with latest commit)